doc/setup/SETUP.md

```diff
@@ -1,22 +1,21 @@
 ## Setup Guide for Artemis
 
 In this guide you learn how to setup the development environment of Artemis.
-Artemis is based on [JHipster](https://jhipster.github.io), i.e. [Spring Boot](http://projects.spring.io/spring-boot) development on the application server using Java 12, 
-and TypeScript development on the application client in the browser using [Angular](https://angular.io) and Webpack. 
-To get an overview of the used technology, have a look at [https://jhipster.github.io/tech-stack](https://jhipster.github.io/tech-stack) and other tutorials on the JHipster homepage.  
+Artemis is based on [JHipster](https://jhipster.github.io), i.e. [Spring Boot](http://projects.spring.io/spring-boot) development on the application server using Java 12, and TypeScript development on the application client in the browser using [Angular](https://angular.io) and Webpack.
```

Comment:
We now use java 14. can you update  this?

Reply:
I missed that one and updated it accordingly. Thanks!